app/pages/History/TransactionDetails.view/TransactionDetails.view.tsx

```diff
@@ -149,7 +148,7 @@ const TransactionDetails: FC<TransactionDetailsViewProps> = ({
           {t('transactionDetails')}
         </Typography>
         <CardContent>
-          {renderRow(`${t('blockHeight')}:`, finalizedBlockIndex)}
+          {renderRow(`${t('blockHeight')}:`, finalizedBlockIndex ?? 'unknown')}
```

Comment:
prevents blank field while the transaction data is being synced, which I had found to be a confusing experience